src/js/utils/index.js

```diff
@@ -27,17 +42,19 @@ export function readLineAsync(query) {
   });
 }
 
-export function calcROI(profit, investmentCost) {
-  return (profit / investmentCost) * 100;
-}
+export async function prompt(
+  query,
+  { validate = (value) => true, format = (value) => value } = {}
```

Comment:
아래처럼 모든 parameter를 객체로 묶지 않고 분리하신 이유가 궁금하네요

```suggestion
  {   query, validate = (value) => true, format = (value) => value } = {}
```

Reply:
`query` parameter는  필수적으로 받기위해 분리하여 작성해보았습니다!

Reply:
필수값, 선택값 여부를 parameter의 구조만 보고는 알 수가 없기 때문에 저는 객체로 모두 받는걸 선호하는 편입니다 ㅎㅎ

Reply:
그렇군요..! 객체로 모두 받도록 변경하였습니다! 감사합니다